app/scripts/controllers/utils/utils.test.js

```diff
@@ -0,0 +1,92 @@
+import { strict as assert } from 'assert';
+import { checkSnapsBlockList } from './utils';
+
+describe('Controller utilities', function () {
```

Comment:
I believe we use arrow functions `() => {}` in tests instead of `function () {}` everywhere else.

Reply:
Yea, we use them everywhere else in snaps repos. But, this one complains if I do that.

```diff
- ESLint: Do not pass arrow functions to describe()(mocha/no-mocha-arrows)
```

Reply:
Roger that